lms/templates/discussion/index.html

```diff
@@ -35,9 +35,9 @@
          data-user-cohort-id="${user_cohort}"
          data-course-settings="${course_settings}">
     <div class="discussion-body">
-        <div class="forum-nav"></div>
-        <div class="discussion-column">
-          <article class="new-post-article" style="display: none"></article>
+        <div class="forum-nav" role="complimentary" aria-label="Discussion thread list"></div>
+        <div class="discussion-column" role="complementary" aria-label="Active Discussion" aria-live="assertive" id="discussion-column">
```

Comment:
see above, I recommended removing the aria-live all together.  It's inappropriate here at all.  Even polite would be a problem.